components/CredentialBase.vue

```diff
@@ -75,10 +75,10 @@
 /*!
  * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
  */
-import {computed, defineProps, reactive, toRef, unref} from 'vue';
```

Comment:
`defineProps` was explicitly left in even though it is not required. There is some more discussion on this here: https://github.com/digitalbazaar/chapi-playground/pull/1#discussion_r894022268

I am unsure if a standard has been chosen for DB on this going forward, but have been including `defineEmits` and `defineProps` as needed to provide clarity on what they are and where they are coming from.

Reply:
Ok, I dropped that commit.  I didn't see that discussion before and was just following the note on the console.  It is very likely people omit this again in the future, here or elsewhere.  I didn't see any rules in https://eslint.vuejs.org/ that would enforce the import and a soft policy won't work so well for this situation.

cc: @dlongley

Reply:
We can continue the discussion here https://github.com/digitalbazaar/chapi-playground/pull/1#discussion_r894022268

Once there is some consensus on the standard going forward I will go ahead and come back and update things if needed. In the meantime we can go ahead and merge this into #1